features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2145,6 +2145,10 @@
             <Permissions>/permission/admin/manage/identity/cors/origins/view</Permissions>
             <Scopes>internal_cors_origins_view</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/configs" secured="true" http-method="GET">
```

Comment:
need to add identity.xml as well

Reply:
Replicated the change in `identity.xml` file in the following commit: 8b45941